app/screens/home/channel_list/additional_tablet_view/additional_tablet_view.tsx

```diff
@@ -34,12 +35,13 @@ const AdditionalTabletView = ({onTeam, currentChannelId, isCRTEnabled}: Props) =
     const [initiaLoad, setInitialLoad] = useState(true);
 
     useEffect(() => {
-        const listener = DeviceEventEmitter.addListener(Navigation.NAVIGATION_HOME, (id: string) => {
+        const listener = DeviceEventEmitter.addListener(Navigation.NAVIGATION_HOME, (id: string, params?: unknown) => {
```

Comment:
Does this work? I was under the impression that it only supported one single param